src/main/java/attendance/model/AttendanceDetail.java

```diff
@@ -0,0 +1,50 @@
+package attendance.model;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class AttendanceDetail {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM월 dd일 EEEE은 등교일이 아닙니다.");
+
+    private LocalDateTime localDateTime;
+    private Attendance attendance;
+
+    public AttendanceDetail(LocalDateTime localDateTime) {
+        validateHoliday(localDateTime);
+        this.localDateTime = localDateTime;
+        this.attendance = Attendance.from(localDateTime);
+    }
+
+    private void validateHoliday(LocalDateTime localDateTime) {
+        if (CustomLocalDateTime.isHoliday(localDateTime.toLocalDate())) {
+            throw new IllegalArgumentException(localDateTime.format(formatter));
+        }
+    }
+
+    public void modify(LocalTime localTime) {
+        localDateTime = LocalDateTime.of(
+                localDateTime.toLocalDate(),
+                localTime
+        );
+        this.attendance = Attendance.from(localDateTime);
+    }
+
+    public Attendance getAttendance() {
+        return attendance;
+    }
+
+    public LocalDate getAttendanceDate() {
+        return localDateTime.toLocalDate();
+    }
+
+    public LocalDateTime getAttendanceDateTime() {
+        return localDateTime;
+    }
+
+    public boolean isSameAs(Attendance attendance) {
+        return attendance.equals(this.attendance);
```

Comment:
보통 Enum 비교는 `equals` 보단 `==` 을 씁니다~

```suggestion
        return attendance == this.attendance;
```

Reply:
수정했습니다! 몰랐던 내용이네요. 

- Enum은 내부적으로 싱글톤 관리이기 때문에 ==를 사용해도 안전하게 비교할 수 있다
- 성능도 equals보다 좋다 (equals는 내부적으로 instanceof 확인 후 비교, ==는 즉시 참조 비교)
- NPE가 발생하지 않는다
- == 비교는 컴파일 타임에 타입 호환성 검사를 지원하기 때문에 프로그래머의 실수를 컴파일 수준에서 빠르게 잡을 수 있다